app.rb

```diff
@@ -0,0 +1,124 @@
+require_relative 'person'
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
```

Comment:
**All requirements are complete! Congratulations 👏👏👏**

- [ ] [OPTIONAL] I would suggest you fix up the lines in your MIT license... As you can see in this image there are some places that need to be changed:

<img width="1035" alt="Screen Shot 2022-04-09 at 2 42 09 AM" src="https://user-images.githubusercontent.com/69011963/162536327-fba6a7a0-a7c3-4c7f-8827-28a1300c81fa.png">

Reply:
Thank you, I'll make sure to fix it.